osbs/api.py

```diff
@@ -405,12 +418,19 @@ def create_prod_build(self, git_uri, git_ref,
         :param architecture: str, build architecture
         :param yum_repourls: list, URLs for yum repos
         :param koji_task_id: int, koji task ID requesting build
+        :param filesystem_koji_task_id: int, koji task for base image filesystem
         :param scratch: bool, this is a scratch build
+        :param platform: str, the platform to build for
```

Comment:
Its not clear why won't `architecture` be reused then. Platforms would not match architectures?